app/views/users/show.html.erb

```diff
@@ -1 +1,3 @@
-<p><%= @specific_user %></p>
\ No newline at end of file
+<%= render "user" %>
+<p><%= @specific_user.bio %></p>
+<p><%= render "post" %></p>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design%20from%20the%20sneak%20peek%20wireframes), Please add basic styling to users show view to make it match the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png).